### PR TITLE
[To rel/0.12][IoTDB-1989] IoTDB support insert data from Spark

### PR DESCRIPTION
**Pull request by xuanronaldo**

### CODE REVIEW

**Review by SilverNarcissus — APPROVED** [review]
Look good to me~

File: spark-iotdb-connector/src/test/scala/org/apache/iotdb/spark/db/IoTDBWriteTest.scala [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      (1L, 1, 1L, 1.0F, 1.0D, true, "hello"),
+      (2L, 2, 2L, 2.0F, 2.0D, false, "world")))
+
+    val dfWithColumn = df.withColumnRenamed("_1", "Time")
+      .withColumnRenamed("_2", "root.test.d0.int")
+      .withColumnRenamed("_3", "root.test.d0.long")
+      .withColumnRenamed("_4", "root.test.d0.float")
+      .withColumnRenamed("_5", "root.test.d0.double")
+      .withColumnRenamed("_6", "root.test.d0.boolean")
+      .withColumnRenamed("_7", "root.test.d0.text")
```
Comment (HTHou) [inline spark-iotdb-connector/src/test/scala/org/apache/iotdb/spark/db/IoTDBWriteTest.scala:80]: Use s0 s1 s2 s3 s4....

File: spark-iotdb-connector/src/test/scala/org/apache/iotdb/spark/db/IoTDBWriteTest.scala [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+      (2L, "root.test.d0", 2, 2L, 2.0F, 2.0D, false, "world")))
+
+    val dfWithColumn = df.withColumnRenamed("_1", "Time")
+      .withColumnRenamed("_2", "device_name")
+      .withColumnRenamed("_3", "int")
+      .withColumnRenamed("_4", "long")
+      .withColumnRenamed("_5", "float")
+      .withColumnRenamed("_6", "double")
+      .withColumnRenamed("_7", "boolean")
+      .withColumnRenamed("_8", "text")
```
Comment (HTHou) [inline spark-iotdb-connector/src/test/scala/org/apache/iotdb/spark/db/IoTDBWriteTest.scala:105]: Better to use s1 s2 s3...

File: spark-iotdb-connector/src/test/scala/org/apache/iotdb/spark/db/IoTDBWriteTest.scala (hunk 1 above)
Comment (xuanronaldo) [reply spark-iotdb-connector/src/test/scala/org/apache/iotdb/spark/db/IoTDBWriteTest.scala:80]: Ok，got this

File: spark-iotdb-connector/src/test/scala/org/apache/iotdb/spark/db/IoTDBWriteTest.scala (hunk 2 above)
Comment (xuanronaldo) [reply spark-iotdb-connector/src/test/scala/org/apache/iotdb/spark/db/IoTDBWriteTest.scala:105]: ok，got this

Comment (xuanronaldo) [reply spark-iotdb-connector/src/test/scala/org/apache/iotdb/spark/db/IoTDBWriteTest.scala]: > Better to use s1 s2 s3...

I modified it and add examples. Review the code, plz.

**Review by HTHou — APPROVED** [review]

**Pull request merged**